demos/build_demos_msvc.bat

```diff
@@ -34,6 +35,14 @@ if not "%1" == "" (
         shift & shift
         goto argParse
     )
+    rem to build more than one specific demo use quotation marks,
+    rem list the necessary demos separated by space,
+    rem ex. --target="classification_demo segmentation_demo"
+    if "%1" == "--target" (
+        set BUILD_TARGETS=%BUILD_TARGETS% %1 %~2
```

Comment:
Again, why are you adding `%1` to `BUILD_TARGETS`?

Reply:
A bit of misunderatanding from my side, fixed it.